.kokoro/tests/build_cloud_functions.sh

```diff
@@ -57,7 +68,7 @@ elif [[ "$file" == *"hello-pubsub"* ]]; then
   gcloud functions deploy $FUNCTIONS_PUBSUB_FN_NAME \
     --region $FUNCTIONS_REGION \
     --runtime $FUNCTIONS_JAVA_RUNTIME \
-    --entry-point "functions.HelloPubSub" \
+    --entry-point "functions.${LANGUAGE}HelloPubSub" \
```

Comment:
I didn't check all of them but [this](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/2990b83f517c3948385f1610a0ead297aeb53d67/functions/helloworld/kotlin-hello-pubsub/src/main/kotlin/functions/KotlinHelloBackground.kt#L27) shows that it's the file name not the entrypoint name that has LANGUAGE prefixed